README.md

```diff
@@ -14,14 +14,14 @@ Terraform configuration used to create the required AWS resources for integratin
 
 | Name      | Version |
 | ----------- | ----------- |
-| [aws](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)      | >= 4.37.0    |
+| [aws](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)      | >= 5.26.0    |
 
 ## Inputs
```

Comment:
There is some missing fields in the documentation. check out this tool https://terraform-docs.io/ it can help you to manage it
`lambda_source_code_path`
`frontend_lambda_source_code_path`
`backend_lambda_source_code_path`
`lambda_logs_retention_in_days`
`lambda_enable_logs`
`resource_name_common_part`
`secrets_names`
`gateway_api_integration_timeout_milliseconds`